TurtleBot3 Note Updates. (#979)
ROBOTIS-GIT/emanual: docs/en/platform/turtlebot3/raspberry_pi_3_setup.md

{% endcapture %}
<div class="notice--warning">{{ notice_01 | markdownify }}</div>

{% capture info_01 %}

> **NOTE**: Use either of 3 ways to install Linux and ROS on Raspberri pI 3

Reviewer: ```suggestion
**NOTE**: Use either of 3 ways to install Linux and ROS on Raspberry Pi 3
```

> 1. For Ubuntu Mate installation, read `Install Linux (Ubuntu MATE)` guide. Be sure to install ROS and dependency packages on your TurtleBot PC. The instruction takes about 1 hours to install ROS and related packages for TurtleBot3.

Reviewer: ```suggestion
1. For Ubuntu Mate installation, read `Install Linux (Ubuntu MATE)` guide. Be sure to install ROS and dependency packages after installing the Linux images on SBC of TurtleBot. The instruction takes about 1 hour to install ROS and related packages for TurtleBot3.
```

2. For Linux distro image installation based on Raspbian, read  `Install Linux (Raspbian)` guide. You do not have to do additional installations as the distro image contains ROS and ROS packages related to TurtleBot3.  
3. For webOS Robotics Platform, read `webOS Robotics Platform` guide. You do not need to compile packages on TurtleBot3. They will be cross-compiled using OpenEmbedded on a higher performance PC, Ubuntu 18.04 based and an image file created from them.
{% endcapture %}
<div class="notice--info">{{ info_01 | markdownify }}</div>

{% capture info_02 %}
**NOTE**: Raspberry Pi 3 B+ is available in TurtleBot3 Burger and Waffle Pi. If you use Raspberry Pi 3 B+, please refer to the following.

> - As Ubuntu MATE does not support the Raspberry Pi B+, do not follow `Install Linux (Ubuntu MATE)` guide. Instead, we recommend to follow `Install Linux based on Raspbian` guide.

Reviewer: ```suggestion
```
Now it's supported. 
(REF. https://ubuntu-mate.org/raspberry-pi/)
